pandas/util/_print_versions.py

```diff
@@ -60,56 +61,54 @@ def show_versions(as_json=False):
     sys_info = get_sys_info()
 
     deps = [
-        # (MODULE_NAME, f(mod) -> mod version)
-        ("pandas", lambda mod: mod.__version__),
-        ("pytest", lambda mod: mod.__version__),
-        ("pip", lambda mod: mod.__version__),
-        ("setuptools", lambda mod: mod.__version__),
-        ("Cython", lambda mod: mod.__version__),
-        ("numpy", lambda mod: mod.version.version),
-        ("scipy", lambda mod: mod.version.version),
-        ("pyarrow", lambda mod: mod.__version__),
-        ("xarray", lambda mod: mod.__version__),
-        ("IPython", lambda mod: mod.__version__),
-        ("sphinx", lambda mod: mod.__version__),
-        ("patsy", lambda mod: mod.__version__),
-        ("dateutil", lambda mod: mod.__version__),
-        ("pytz", lambda mod: mod.VERSION),
-        ("blosc", lambda mod: mod.__version__),
-        ("bottleneck", lambda mod: mod.__version__),
-        ("tables", lambda mod: mod.__version__),
-        ("numexpr", lambda mod: mod.__version__),
-        ("feather", lambda mod: mod.__version__),
-        ("matplotlib", lambda mod: mod.__version__),
-        ("openpyxl", lambda mod: mod.__version__),
-        ("xlrd", lambda mod: mod.__VERSION__),
-        ("xlwt", lambda mod: mod.__VERSION__),
-        ("xlsxwriter", lambda mod: mod.__version__),
-        ("lxml.etree", lambda mod: mod.__version__),
-        ("bs4", lambda mod: mod.__version__),
-        ("html5lib", lambda mod: mod.__version__),
-        ("sqlalchemy", lambda mod: mod.__version__),
-        ("pymysql", lambda mod: mod.__version__),
-        ("psycopg2", lambda mod: mod.__version__),
-        ("jinja2", lambda mod: mod.__version__),
-        ("s3fs", lambda mod: mod.__version__),
-        ("fastparquet", lambda mod: mod.__version__),
-        ("pandas_gbq", lambda mod: mod.__version__),
-        ("pandas_datareader", lambda mod: mod.__version__),
-        ("gcsfs", lambda mod: mod.__version__),
+        "pandas",
+        "pytest",
+        "pip",
+        "setuptools",
+        "Cython",
+        "numpy",
+        "scipy",
+        "pyarrow",
+        "xarray",
+        "IPython",
+        "sphinx",
+        "patsy",
+        "dateutil",
```

Comment:
can you only list the ones that are *not* in pandas.compat._optional (and just import from that list)?